easybuild/easyblocks/p/python.py

```diff
@@ -640,7 +638,11 @@ def make_module_extra(self, *args, **kwargs):
         """Add path to sitecustomize.py to $PYTHONPATH"""
         txt = super(EB_Python, self).make_module_extra()
 
+        # Legacy support for existing installations doing "--rebuild --module-only"
         if self.cfg.get('ebpythonprefixes'):
-            txt += self.module_generator.prepend_paths(PYTHONPATH, self.pythonpath)
+            new_dir = os.path.join('lib', 'python' + self.pyshortver, 'site-packages')
+            old_dir = os.path.join(log_path(), 'python')
+            if not os.path.exists(os.path.join(self.installdir, new_dir, 'sitecustomize.py')):
```

Comment:
Shall we maybe use a "constant" for this path similar to `self.pythonpath` to avoid repeating it in 2 locations that are quite a bit apart? I'd also check if the script exists in `old_dir` (or at least that folder) before adding the old path